apps-rendering/src/components/emailSignup.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import {
+	background,
+	remSpace,
+	text,
+	textSans,
+} from '@guardian/source-foundations';
+import type { EmailSignup } from 'embed';
+import { maybeRender } from 'lib';
+import type { FC } from 'react';
+import { darkModeCss } from 'styles';
+
+interface Props {
+	embed: EmailSignup;
+}
+
+const styles = css`
+	margin: ${remSpace[4]} 0;
+
+	${darkModeCss`
+        background: ${background.inverse};
```

Comment:
I'm a bit unsure about this. For `GenericEmbed`, this is set to `white` (https://github.com/guardian/dotcom-rendering/pull/3732/files#diff-b59bf599937aca257cb6d5e80b102b71a4dc7dabef64bf46b47ec2e5acfc170eR17)

Reply:
I'll revert this, so we can spend the time looking at it in the context of design & accessibility requirements